### ISSUE TITLE
Allow routes to insert new custom element without import

### ISSUE THREAD

**Issue by gazal-k** [Plastikit/plastik-lazy-route-selector#2]
I see that plastik-lazy-route-selector is inspired by app-router. app-router allowed for route configurations like the following:

``` html
    <plastik-lazy-route-selector main selected="{{route}}">
      <iron-pages attr-for-selected="route">
        <div route="app-login" element="app-login"></div>
        <div route="dashboards-screen" element="main-screen"></div>
      </iron-pages>
    </plastik-lazy-route-selector>
```

If custom elements with name app-login and main-screen have already been registered, then it should work without an HTML import.
But with plastik-lazy-route-selector, the import attribute seems mandatory.

**Comment by adalinesimonian** [comment]
I agree this should be a handled case. Thanks for the PR! I'm going to go take a look at it and give it a test drive. :+1:

**Comment by adalinesimonian** [comment]
Merged #3. Thanks!

**Comment by gazal-k** [comment]
:+1:
